### ISSUE TITLE
Address missing alternative text accessibility findings in cloud.gov Pages public site

### ISSUE THREAD

**Issue by svenaas** [cloud-gov/cg-site#2302]
Address missing alternative text accessibility findings in cloud.gov Pages public site. Alt attributes are missing on:

Fact sheet thumbnails on https://cloud.gov/pages/:

- Pages proposal
- Pages basic diagram
- Compliance Memo and Extension

UI screenshots on https://cloud.gov/pages/documentation/access-permissions/:

- Adding a user to an organization: Your Organizations
- Adding a user to an organization: Pages Support

**Comment by svenaas** [comment]
Findings:

### Missing alternative text

1. https://cloud.gov/pages/

Missing alternative text on these images:

* Pages proposal
* Pages basic diagram
* Compliance Memo and Extension

2. https://cloud.gov/pages/documentation/access-permissions/

Missing alternative text on these images:

* Adding a user to an organization: Your Organizations
* Adding a user to an organization: Pages Support

### Scanned; no WAVE errors (apart from contrast, listed separately in #2303)

1. https://cloud.gov/pages/features/
2. https://cloud.gov/pages/success-stories/
3. https://cloud.gov/pages/pricing/
4. https://cloud.gov/pages/contact/
5. ~~https://cloud.gov/pages/federalist-migration/~~ (removed in #2299)
6. https://cloud.gov/pages/documentation/
7. https://cloud.gov/pages/documentation/why-use-pages/
8. https://cloud.gov/pages/documentation/adding-users/
9. https://cloud.gov/pages/documentation/customer-responsibilities/
10. https://cloud.gov/pages/documentation/sandbox/
11. https://cloud.gov/pages/documentation/21st-century-idea/
12. https://cloud.gov/pages/documentation/getting-started-with-jekyll/
13. https://cloud.gov/pages/documentation/getting-started-with-netlify-cms/
14. https://cloud.gov/pages/documentation/included-with-pages/
15. https://cloud.gov/pages/documentation/instructional-demos/
16. https://cloud.gov/pages/documentation/previews/
17. https://cloud.gov/pages/documentation/custom-domains/
18. https://cloud.gov/pages/documentation/before-you-launch/
19. https://cloud.gov/pages/documentation/migration-guide/
20. https://cloud.gov/pages/documentation/forms/
21. https://cloud.gov/pages/documentation/search/
22. https://cloud.gov/pages/documentation/templates/
23. https://cloud.gov/pages/documentation/customization/
24. https://cloud.gov/pages/documentation/resources/
25. https://cloud.gov/pages/documentation/content-guide/
26. https://cloud.gov/pages/documentation/custom-headers/
27. https://cloud.gov/pages/documentation/federalist-json/
28. https://cloud.gov/pages/documentation/how-builds-work/
29. https://cloud.gov/pages/documentation/build-errors/
30. https://cloud.gov/pages/documentation/env-vars-on-pages-builds/
31. https://cloud.gov/pages/documentation/cloud-gov/
32. https://cloud.gov/pages/documentation/node-on-pages/
33. https://cloud.gov/pages/documentation/rvm-on-pages/
34. https://cloud.gov/pages/documentation/bundler-on-pages/
35. https://cloud.gov/pages/documentation/cache-dependencies/
36. https://cloud.gov/pages/documentation/supported-site-engines/
37. https://cloud.gov/pages/documentation/large-file-handling/
38. https://cloud.gov/pages/documentation/renaming-site-repository/
39. https://cloud.gov/pages/documentation/monorepos-on-pages/
40. https://cloud.gov/pages/documentation/external-tools-and-resources/

**Comment by svenaas** [comment]
Resolved with cloud-gov/cg-site#2311
